Lesson 18/task-1.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+VALID_EMAILS = ['abc-d@mail.com', 'abc.def@mail.com', 'abc@mail.com', 'abc_def@mail.com',
```

Comment:
I encourage you to use formatters like black or yapf. Default IDE's formatting s*cks.